js/bibox.js

```diff
@@ -220,17 +220,15 @@ module.exports = class bibox extends Exchange {
     }
 
     parseTrade (trade, market = undefined) {
-        let timestamp = trade['time'];
-        let side = this.safeInteger (trade, 'side');
-        side = this.safeInteger (trade, 'order_side', side);
+        let timestamp = trade['createdAt'];
```

Comment:
You did break the parsing for fetchTrades with this edit. Can you make parseTrade parse both the public and private trades, like it does for other exchanges? ) Thx!

Reply:
A public trade looks like this:

```
              {     id:  35131874,
                 pair: "ETH_BTC",
                price: "0.07678341",
               amount: "0.0873",
                 time:  1528748288000,
                 side:  2              },
```

Reply:
Thanks for the input. Please take a look at the latest

Reply:
btw, how did you get this dict to print right-justified like this?

Reply:
@benediamond using [string.ify](https://github.com/xpl/string.ify) (https://www.npmjs.com/package/string.ify) + [ololog](https://github.com/xpl/ololog) (https://www.npmjs.com/package/ololog):

```
mbp:ccxt igorkroitor$ node examples/js/cli kraken market ETH/BTC --load-markets
kraken.market (ETH/BTC)
{     limits: { amount: { min: 0.02, max: 100000000 },
                 price: { min: 0.00001, max: undefined },
                  cost: { min: 0, max: undefined }        },
   precision: { amount: 8, price: 5 },
   tierBased:    true,
  percentage:    true,
       taker:    0.0026,
       maker:    0.0016,
       tiers: { taker: [ [0, 0.0026],
                         [50000, 0.0024],
                         [100000, 0.0022],
                         [250000, 0.002],
                         [500000, 0.0018],
                         [1000000, 0.0016],
                         [2500000, 0.0014],
                         [5000000, 0.0012],
                         [10000000, 0.0001] ],
                maker: [ [0, 0.0016],
                         [50000, 0.0014],
                         [100000, 0.0012],
                         [250000, 0.001],
                         [500000, 0.0008],
                         [1000000, 0.0006],
                         [2500000, 0.0004],
                         [5000000, 0.0002],
                         [10000000, 0]      ]  },
          id:   "XETHXXBT",
      symbol:   "ETH/BTC",
        base:   "ETH",
       quote:   "BTC",
      baseId:   "XETH",
     quoteId:   "XXBT",
    darkpool:    false,
        info: {             altname:   "ETHXBT",
                        aclass_base:   "currency",
                               base:   "XETH",
                       aclass_quote:   "currency",
                              quote:   "XXBT",
                                lot:   "unit",
                      pair_decimals:    5,
                       lot_decimals:    8,
                     lot_multiplier:    1,
                       leverage_buy: [2, 3, 4, 5],
                      leverage_sell: [2, 3, 4, 5],
                               fees: [ [0, 0.26],
                                       [50000, 0.24],
                                       [100000, 0.22],
                                       [250000, 0.2],
                                       [500000, 0.18],
                                       [1000000, 0.16],
                                       [2500000, 0.14],
                                       [5000000, 0.12],
                                       [10000000, 0.1]  ],
                         fees_maker: [ [0, 0.16],
                                       [50000, 0.14],
                                       [100000, 0.12],
                                       [250000, 0.1],
                                       [500000, 0.08],
                                       [1000000, 0.06],
                                       [2500000, 0.04],
                                       [5000000, 0.02],
                                       [10000000, 0]    ],
                fee_volume_currency:   "ZUSD",
                        margin_call:    80,
                        margin_stop:    40                 },
     altname:   "ETHXBT",
      active:    true                                         }
mbp:ccxt igorkroitor$
```

https://github.com/ccxt/ccxt/tree/master/examples/js/cli.js

Reply:
interesting... I'd like to find a way to output strings like this from within python.